airbyte-commons-worker/src/main/java/io/airbyte/workers/helper/FailureHelper.java

```diff
@@ -113,6 +113,13 @@ public static FailureReason sourceFailure(final Throwable t, final Long jobId, f
         .withExternalMessage("Something went wrong within the source connector");
   }
 
+  public static FailureReason sourceHeartbeatFailure(final Throwable t, final Long jobId, final Integer attemptNumber) {
+    return connectorCommandFailure(t, jobId, attemptNumber, ConnectorCommand.READ)
+        .withFailureOrigin(FailureOrigin.SOURCE)
+        .withFailureType(FailureType.HEARTBEAT_TIMEOUT)
+        .withExternalMessage("The source was unresponsive");
```

Comment:
Nit:  avoid passive tense -> use "is" instead of "was".

Reply:
done